.github/workflows/check-if-migration-is-required.yml

```diff
@@ -47,3 +47,5 @@ jobs:
         run: |
           # Below will exit with non-zero status if model changes are missing migrations
           sentry django makemigrations -n ci_test --check --dry-run --no-input || (echo '::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name>`' && exit 1)
+          [ $(git status --short | grep "M migrations_lockfile.txt" | wc -l) == 1 ] && \
+            echo >&2 "::error::Error: You have not updated the migration_lockfile.txt" && exit 1
```

Comment:
Did we want to try grab the latest migration filename and make sure it's in the lockfile? As-is is probably enough, but that would add one extra level of safety

Reply:
It would be more work but it's doable. We could even create a pre-commit hook based on it.